docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: "3.8"
+services:
+  web:
+    image: crystallang/crystal
```

```suggestion
    image: crystallang/crystal:0.35.1
```

Add version so it's not latest implicitly.